package.json

```diff
@@ -45,6 +45,7 @@
     "lerna-changelog": "^0.8.2",
     "pretty-quick": "^1.10.0",
     "ts-jest": "^24.0.2",
+    "ts-node": "^8.1.0",
```

Comment:
why is ts-node needed in the root package.json?

Reply:
I added the ts-node dependency for debugging purposes. As the package is written in typescript, in order to debug it in vscode and not build it first, you need to run any lerna package you want to debug using the ts-node module.